build.sbt

```diff
@@ -123,8 +124,8 @@ lazy val scala = Project(id = "content-atom-model", base = file("scala"))
       (scroogeUnpackDeps in Compile).value.flatMap { dir => (dir ** "*.thrift").get }
     },
     libraryDependencies ++= Seq(
-      "org.apache.thrift" % "libthrift" % "0.9.2",
-      "com.twitter" %% "scrooge-core" % "4.5.0"
+      "org.apache.thrift" % "libthrift" % "0.9.3",
+      "com.twitter" %% "scrooge-core" % "4.12.0"
```

Comment:
no merge as it is. I think we should just keep the downgrade we currently have in [models](https://github.com/guardian/content-api-models/blob/master/build.sbt#L124)